packages/flutter/test/cupertino/refresh_test.dart

```diff
@@ -373,6 +374,76 @@ void main() {
       },
     );
 
+    testWidgets(
+      'refreshing task keeps the sliver expanded forever until completes with error',
+      (WidgetTester tester) async {
+        debugDefaultTargetPlatformOverride = TargetPlatform.iOS;
+        final FlutterError error = FlutterError('Oops');
+        double errorCount = 0;
+
+        runZoned(() async {
+          refreshCompleter = Completer<void>.sync();
+
+          await tester.pumpWidget(
+            Directionality(
+              textDirection: TextDirection.ltr,
+              child: CustomScrollView(
+                slivers: <Widget>[
+                  CupertinoSliverRefreshControl(
+                    builder: builder,
+                    onRefresh: onRefresh,
+                  ),
+                  buildAListOfStuff(),
+                ],
+              ),
+            ),
+          );
+
+          await tester.drag(find.text('0'), const Offset(0.0, 150.0), touchSlopY: 0);
+          await tester.pump();
+          // Let it start snapping back.
+          // Reaches refresh state and sliver's at 60.0 in height after a while.
+          await tester.pump(const Duration(seconds: 1));
+          verify(mockHelper.builder(
+              any,
+              RefreshIndicatorMode.refresh,
+              60.0,
+              100.0, // Default value.
+              60.0, // Default value.
+          ));
+
+          clearInteractions(mockHelper);
+          // Stays in that state forever until future completes with error.
+          await tester.pump(const Duration(seconds: 1000));
+          verifyNoMoreInteractions(mockHelper);
```

Comment:
Ah I see. How many unverified things were there left against the mock? It's more idiomatic to just verify them all if there aren't too many

Reply:
I basically just copied the one test before this and modified a bit. The removed `verify`s are already tested in that test and I felt they were just noises.